File: L018/dispArr.java
import java.util.Arrays;
import java.util.Scanner;

public class dispArr {
    
    public static Scanner sc = new Scanner(System.in);
    public static void main(String[] args) throws Exception {
        // write your code here
        
        int n = sc.nextInt();
        
        int[] arr = new int[n];
        for(int i=0;i<arr.length;i++){
            arr[i] = sc.nextInt();
        }
        
        displayArr(arr,0);
        
    }

    public static void displayArr(int[] arr, int idx) {
        
        if(idx == arr.length){
            return;
        }
        
        System.out.println(arr[idx]);
        
        displayArr(arr,idx+1);

    }

}

File: L018/dispArr.cpp
#include<iostream>
#include<vector>
using namespace std;


 void displayArr(vector<int>& arr, int idx) {
        
        if(idx == arr.size()){
            return;
        }
        
        cout<<arr[idx]<<endl;
        
        displayArr(arr,idx+1);

    }
int main() {
        // write your code here
        
        int n;
        cin>>n;
        
        vector<int>arr(n,0);
        for(int i=0;i<arr.size();i++){
            cin>>arr[i];
        }
        
        displayArr(arr,0);

        return 0;
        
    }

   